runio.js

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+const { stopOnFail, chdir, git, copy, exec, replaceInFile, npmRun, npx, writeToFile, runio } = require('runio');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace. (object-curly-newline)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4496293)